.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,67 +1,52 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: File a bug report
+about: ... if you experienced behavior that you did not expect
 title: ''
 labels: possible bug
 assignees: ''
 
 ---
 
 <!--
-Thank you for filing a bug within privacyIDEA.
-Please read the following blog post.
-https://www.privacyidea.org/getting-help/
+Thank you for filing a bug within privacyIDEA. The more information you provide, the easier it is to identify the problem!
 
-The more information you provide, the easier it is to identify 
-the problem!
+Issues are not meant for asking questions! You're welcome to ask questions in our community forum: https://community.privacyidea.org
 
-Issues are not meant for asking questions!
-If you simply wonder how things work and need some explanation 
-please visit the community forum:
-
-https://community.privacyidea.org
-
-If you do not like the NO-WARRANTY of this software and need
-reliable, professional help, please consider getting either
+privacyIDEA is licensed under the AGPL and comes without warranty. If you require reliable, professional help, please consider getting either
 - consultancy 
   (https://netknights.it/en/leistungen/one-time-services/)
 - or the privacyIDEA Enterprise Edition 
   (https://netknights.it/en/leistungen/service-level-agreements/)
-
-If you still think, you should open an issue, please do so:
 -->
 
-# What did you try to do?
-
-Please provide your top level intent!
-Please provide steps to reproduce!
-
-# What outcome did you expect?
+### Top-level intent
```

Comment:
Hm, here it is - the top level intent. I think I got lost in the github diff-view.

Reply:
Yup, it apparently just slipped away a bit :)